spec/integration/aws_ebs_volume_spec.rb

```diff
@@ -9,11 +9,18 @@
       it "aws_ebs_volume 'test_volume' creates a key pair" do
         expect_recipe {
           aws_ebs_volume "test_volume" do
-            availability_zone 'eu-west-1a'
+            availability_zone "#{driver.aws_config.region}a"
             size 8
           end
         }.to create_an_aws_ebs_volume('test_volume')
          .and be_idempotent
+
+        expect_recipe {
+          aws_ebs_volume "test_volume" do
```

Comment:
This test shouldn't be passing right now.  You should need something like:
```ruby
aws_ebs_volume "test_volume" do
  availability_zone "a"
end
aws_ebs_volume "test_volume" do
  action :destroy
end
```

Otherwise there is no resource to destroy and the matcher should be failing